dist/dicom/dist/demo/app.js

```diff
@@ -1,4 +1,4 @@
-import { structuredReportToText } from '../itk-dicom.js'
+import { structuredReportToHtml } from '../itk-dicom.js'
```

Comment:
I can work on adding support for both in a followup.

For now this should either not be changed, or the test configuration also needs to updated:

https://github.com/InsightSoftwareConsortium/itk-wasm/blob/3efaf52f2fd2a42d0175e0510116d9025156940b/dist/dicom/cypress/e2e/structured_report_to_text.cy.ts

Reply:
Ok, I can leave it for you to update later then. I will remove the corresponding commit.